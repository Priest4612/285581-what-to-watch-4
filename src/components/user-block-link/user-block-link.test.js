import React from 'react';
import renderer from 'react-test-renderer';

import UserBlockLink from './user-block-link.jsx';


it(`Render UserBlockLink`, () => {
  const tree = renderer
    .create(<UserBlockLink />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
