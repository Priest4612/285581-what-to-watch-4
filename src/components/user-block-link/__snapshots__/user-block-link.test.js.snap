// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render UserBlockLink 1`] = `
<a
  className="user-block__link"
  href="sign-in.html"
>
  Sign in
</a>
`;
